test/tests.js

```diff
@@ -63,6 +63,21 @@ QUnit.test('malformed cookie value in IE', function (assert) {
 	document.body.appendChild(iframe);
 });
 
+// github.com/js-cookie/js-cookie/pull/171
+QUnit.test('missing leading semicolon', function (assert) {
+	assert.expect(1);
+	var done = assert.async();
+	// Sandbox in an iframe so that we can poke around with document.cookie.
+	var iframe = document.createElement('iframe');
+	iframe.src = 'missing_semicolon.html';
+
+	addEvent(iframe, 'load', function () {
+		assert.strictEqual(iframe.contentWindow.ok, true, 'encounters `(intermediate value)...` error');
```

Comment:
Since we are not simulating the `(intermediate value)` error specifically, what about creating a more specific message like "can't throw Object is not a function error"?